### ISSUE TITLE
Can't display a long list of DateObject or Quantity objects

### ISSUE THREAD

**Issue by JerryI** [JerryI/wolfram-js-frontend#278]
### Discussed in https://github.com/JerryI/wolfram-js-frontend/discussions/277

<div type='discussions-op-text'>

<sup>Originally posted by **Hachann** September  5, 2024</sup>
Hello,

`RandomDate[n]` outputs a list of Dates (`DateObject` Head) up to n = 147. Starting at n = 148, there is no output.

 `Differences @ RandomDate[n]` outputs a list of dates differences (`Quantity` Head) up to n = 206. Starting at n = 207, there is no output.</div>

**Comment by JerryI** [comment]
In general this is correct. We have probably seen something like this

![Screenshot 2024-09-05 at 13 03 43](https://github.com/user-attachments/assets/4ee8d566-420b-4731-8616-9b4c2f78bf50)

This is natural limitations of big outputs. If we allow bigger, then our editor (CodeMirror 6) struggles to render all of it. In order not to affect UX, we limit the maximum output length (as characters).

**Comment by Hachann** [comment]
I don't get to output the "Too long to display" mention you seem to have, which is nice actually, but either an error message or sometimes even nothing. The computation is done and the data is retrievable for later transformations.
![image](https://github.com/user-attachments/assets/b753c31d-d025-4e03-aee6-1bf19c6cddd8)

**Comment by JerryI** [comment]
> I don't get to output the "Too long to display" mention you seem to have, which is nice actually, but either an error message or sometimes even nothing. The computation is done and the data is retrievable for later transformations. ![image](https://private-user-images.githubusercontent.com/22450300/364752648-b753c31d-d025-4e03-aee6-1bf19c6cddd8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1Mzg2NzcsIm5iZiI6MTcyNTUzODM3NywicGF0aCI6Ii8yMjQ1MDMwMC8zNjQ3NTI2NDgtYjc1M2MzMWQtZDAyNS00ZTAzLWFlZTYtMWJmMTljNmNkZGQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDEyMTI1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5NDAzMTNiYjczMTA0OGMyZDNmM2Q4NTEzNWNhMjVmYmYxMzU1ODg3MDNlZDQ0MDkzY2MwMDJmMGJhYmEyNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UH7WQuFRgC__3lR2gi6Mw9yOHoCTnLAAHXhRdN0X4A4)

I see. Now the differences between linux Chromium browser and windows/osx kicks in.
Try to `Menu -> Reload Window`

**Comment by JerryI** [comment]
> I don't get to output the "Too long to display" mention you seem to have, which is nice actually, but either an error message or sometimes even nothing. The computation is done and the data is retrievable for later transformations. ![image](https://private-user-images.githubusercontent.com/22450300/364752648-b753c31d-d025-4e03-aee6-1bf19c6cddd8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU2MTQ3NjIsIm5iZiI6MTcyNTYxNDQ2MiwicGF0aCI6Ii8yMjQ1MDMwMC8zNjQ3NTI2NDgtYjc1M2MzMWQtZDAyNS00ZTAzLWFlZTYtMWJmMTljNmNkZGQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA2VDA5MjEwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4NmE5NWEwMTE1NGVhMmNiYzQ0M2U3MjRmNTg0Njc1OTk0MmNlYzkzNjdlOGFkODc2NTQyNTAyMTk1Mzg0YmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.F0jmDFxbxqA0swyBGaVxlLL-_fftAQXByaDf-tQFrOs)

I still could not reproduce it on my Linux machine. In a case if something similar shows up `new window.CellWrapper`, could you please share logs with us? 

- Right click -> Inspect
- Console

**Comment by Hachann** [comment]
Sure, this is the error message : 

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'view')
    at new window.CellWrapper (module.js:465:64)
    at core.UINotebook.NewCell (/home/hachann/Wolfram Language/wln notebooks/start.wln:1350:17)
```

**Comment by Hachann** [comment]
Is this verbose enough ?

![image](https://github.com/user-attachments/assets/d4a940ff-6f98-4c19-a0d8-8689b1de6610)

**Comment by JerryI** [comment]
> > I don't get to output the "Too long to display" mention you seem to have, which is nice actually, but either an error message or sometimes even nothing. The computation is done and the data is retrievable for later transformations. ![image](https://private-user-images.githubusercontent.com/22450300/364752648-b753c31d-d025-4e03-aee6-1bf19c6cddd8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1Mzg2NzcsIm5iZiI6MTcyNTUzODM3NywicGF0aCI6Ii8yMjQ1MDMwMC8zNjQ3NTI2NDgtYjc1M2MzMWQtZDAyNS00ZTAzLWFlZTYtMWJmMTljNmNkZGQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDEyMTI1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5NDAzMTNiYjczMTA0OGMyZDNmM2Q4NTEzNWNhMjVmYmYxMzU1ODg3MDNlZDQ0MDkzY2MwMDJmMGJhYmEyNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UH7WQuFRgC__3lR2gi6Mw9yOHoCTnLAAHXhRdN0X4A4)
> 
> I see. Now the differences between linux's Chromium browser and Windows/OSX kick in. Try to `top Menu -> Reload Window`

If you do this on the page with errors, does it change anything?

**Comment by JerryI** [comment]
Resolved in a 2.5.6 release
